google/ggrc-core
# GGRC-1408/GGRC-1351 Rename 'status' to 'state' for audit

# Issue description

Rename "Status" to "State" in Audit csv file, Export page and tree view.

# Steps to test the changes

1. On the My work page open Audit tab
2. Look at the state column: is shown as "status"

**Actual Result:** "Status" column is shown for audit
**Expected Result:** "State" column is shown for audit

1. Log as Admin and go to Data Import page
2. Upload csv file for Audit

**Actual Result:** "Status" field is displayed in csv file for Audit
**Expected Result:** "State" field is displayed in csv file for Audit

**Note:** Also "Status" field should be renamed in Data Export page

# Solution description

Update wording.

# Sanity checklist




# PR Review checklist
